opentutor_classifier/opentutor_classifier/lr/clustering_features.py

```diff
@@ -168,14 +156,12 @@ def generate_patterns_from_candidates(
 
     def generate_feature_candidates(
         self,
-        good_answers: np.array,
-        bad_answers: np.array,
+        good_answers: np.ndarray,
```

Comment:
function needs return type

Reply:
Actually, this function can have multiple return types. and If I include set return type as Union then it causes multiple type errors in other functions.